media-gfx/shotwell/shotwell-0.30.4.ebuild

```diff
@@ -1,7 +1,7 @@
 # Copyright 1999-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
 VALA_MIN_API_VERSION="0.40"
 VALA_MAX_API_VERSION="0.42"
```

Comment:
Does it perhaps support vala:0.44 or 0.46 too now? Maybe there was a NEWS item for this, or it works if you build it with 0.44?

Reply:
Seems to build just fine with 0.46!